Bottom: 6bf109942aac5bb94cbf0138ade6657d9b8eb2b5
Top:    c63bd9504a6634ca6b6e9860759a4d2ae4202202
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-26 13:04:46 -0500

Refresh of refresh-temp

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index d9c2de43..a84f4040 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -162,7 +162,6 @@
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
     hostname: '{{ container_addr_map.mantisbt.hostname }}'
-    hostname: '{{ mantisbt_hostname }}'
     ports: '{{ container_port_map.mantisbt.port_args }}'
     link: 
       -'{{ openldap_container_name }}:mantisbt_ldap'
